.github/workflows/push.yaml

```diff
@@ -0,0 +1,19 @@
+name: GitHub Actions
+on: [push]
+env:
+  DOCKERHUB_USER: ${{ secrets.DOCKERHUB_USER }}
+  DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Build image
+        run: |
+          docker login --username "$DOCKERHUB_USER" --password "$DOCKERHUB_TOKEN"
+          docker-compose build
+      - name: Install composer dependencies
+        run: docker-compose run dev composer install
```

Comment:
tady asi není potřeba docker-compose nebo jo?

Reply:
jo to asi není, opravím 👍

Reply:
opraveno